apis/src/websocket/client_handlers/response_handler.rs

```diff
@@ -5,14 +5,13 @@ use super::{
     tournament::handler::handle_tournament, user_search::handle::handle_user_search,
     user_status::handle::handle_user_status,
 };
-use crate::common::{CommonMessage, ServerMessage::*, ServerResult};
+use crate::common::{ServerMessage::*, ServerResult, WebsocketMessage};
 use leptos::logging::log;
-use leptos::*;
 
-pub fn handle_response(m: &CommonMessage) {
-    batch(move || match m {
-        CommonMessage::Server(result) => match result {
-            ServerResult::Ok(message) => match *message.clone() {
+pub fn handle_response(m: WebsocketMessage) {
+    match m {
```

Comment:
The batch here was so that when one of the handle_...updates multiple signals it doesn't trigger multiple updates and it just updates once for all of them.

Reply:
Ive put it back. To note on leptos 0.7 everything is batched and the api is gone, so they will need to be removed eventually